src/custom-elements/src/components/dropdown/dropdown.tsx

```diff
@@ -348,6 +370,12 @@ export class Dropdown {
           ${this.fluid ? FLUID_CLASS : ''}
           ${this.animateChevron ? ANIMATE_CLASS : ''}
         `}
+        color={`${this.color || ''}`}
+        variant={`${this.variant || ''}`}
+        size={`${this.size || ''}`}
+        uppercase={this.uppercase}
+        center={this.center}
```

Comment:
@ASanchLumen this property doesn't exist in dropdown. Jenkins has failed. Could you remove it?